src/meltano/core/tracking/tracker.py

```diff
@@ -268,6 +268,7 @@ def track_struct_event(self, category: str, action: str) -> None:
                 category=category,
                 action=action,
                 label=str(self.project_id),
+                context=self.contexts,
```

Comment:
Instead of passing the contexts in when the legacy tracker is initialized, could we pass in the Snowplow tracker, and then have this take whatever the current contexts are?

Reply:
I might be misunderstanding what you're asking, can you clarify/explain why? I could see having Tracker() accept an initialized Snowplow tracker make sense, but having Tracker() accept a snowplow tracker, to only use its contexts when firing a structured event seems odd?

Having a structured event always include its trackers context's seems like a good practice in general, its how I'd expect `Tracker().track_struct_event()` to behave, because that's also how `Tracker().track_unstruct_event()` behaves.

Reply:
My apologies. I misunderstood what this change was doing. Please ignore my comment. I probably shouldn't be reviewing these PRs until I get home.

Reply:
No worries! I'm super grateful you're around to 👀, even in passing. It's probably far too late for me to be writing quality stuff 😁 I'm definitely on the down slope after the last few days.